internal/upgrader/upgrader_test.go

```diff
@@ -181,55 +181,84 @@ var _ = Describe("Upgrade", func() {
 	})
 
 	When("running with --check-up-to-date", func() {
-		It("should print out service GUIDs and not attempt to upgrade", func() {
-			result := captureStdout(func() {
-				l := logger.New(100 * time.Millisecond)
-				defer l.Cleanup()
-				err := upgrader.Upgrade(fakeCFClient, l, upgrader.UpgradeConfig{
-					BrokerName:       fakeBrokerName,
-					ParallelUpgrades: 5,
-					CheckUpToDate:    true,
-				})
-				Expect(err).To(MatchError("check up-to-date failed: found 3 instances which are not up-to-date"))
-			})
+		Context("there are instances not up to date", func() {
```

Comment:
nitpick(non-blocking): `not up to date` sounds convoluted.
```suggestion
		Context("there are outdated instances", func() {
```